finalProject/highLevelCC.md

```diff
@@ -0,0 +1,6 @@
+The first thing I would do is make a ceasarCipher class in my js.   
+I would then make the class hold a cipher() method holding a string.   
+Then add an alphabet var in all caps, since it doesnt matter much to us yet.
+Then adjust the shift to 26.   
+Then make an if else statement iterating over the data. Also going through special characters.   
```

Comment:
Will you be doing any process conditionally? Or will you do the same process for every letterr?